### PR TITLE
New File Added:Assignment 9 

### PR DESCRIPTION
**Pull request by Bhavika222**

### CODE REVIEW

**Review by Akash-ing — APPROVED** [review]

File: Bhavika/html/Assignment 9/Assi9.style.css [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+    background-color: purple;
+    color: white;
+} */
+.img6 {
+    position: absolute;
+    height: 231px;
+    width: 362px;
+    left: 879px;
+    top: 490px;
+}
```
Comment (Pawan1005) [inline Bhavika/html/Assignment 9/Assi9.style.css:316]: remove commented code

File: Bhavika/html/Assignment 9/Assi9.html [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+                            <h5 class="kolkata">KOLKATA</h5>
+                            <h5 class="delhi">DELHI</h5>
+                            <h5 class="ahemdabad">AHEMDABAD</h5>
+                            <button class="viewbtn">VIEW NATIONAL</button>
+                            </div> -->
+                            <div>
+                            <img src="m.png" alt="bestconsultant" class="img6">
+                            </div>
+    </body>
+</html>
```
Comment (Pawan1005) [inline Bhavika/html/Assignment 9/Assi9.html:110]: remove commented code

File: Bhavika/html/Assignment 9/Assi9.html [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+                    </div>
+                    <div>
+                        <img src="user.gif" alt="background image" class="img4">
+                        
+                    </div>
+                    
+                    <section>
+                        <h4 class="h4">BROWSE JOBS</h4>
+                                <div class="z1">
+                                    <h5>Information Technology</h5>
```
Comment (Pawan1005) [inline Bhavika/html/Assignment 9/Assi9.html:81]: make correct indentation

File: Bhavika/html/Assignment 9/Assi9.html [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+                                </div>
+                         </div>
+                    <div>
+                        <img src="22.png" alt="background image" class="img2">
+                    </div>
+                    <div>
+                        <img src="user.gif" alt="background image" class="img4">
+                        
+                    </div>
+
```
Comment (Pawan1005) [inline Bhavika/html/Assignment 9/Assi9.html]: remove extra lines

File: Bhavika/Bootstrap/person.svg [hunk 5]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="#000000"><path d="M0 0h24v24H0V0z" fill="none"/><path d="M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z"/></svg>
```
Comment (Pawan1005) [inline Bhavika/Bootstrap/person.svg:1]: add new line to end of  every file

File: Bhavika/Bootstrap/bootstrap.min.css.xml [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+       style="fill:#ffffff" />
+    <path
+       d="M52,37.05a1.5,1.5,0,0,0-1.5,1.5v12h-12a1.5,1.5,0,0,0,0,3H52a1.51,1.51,0,0,0,1.5-1.5V38.55A1.5,1.5,0,0,0,52,37.05Z"
+       id="path6"
+       style="fill:#ffffff" />
+    <path
+       d="M25.45,50.52h-12v-12a1.5,1.5,0,0,0-3,0V52a1.51,1.51,0,0,0,1.5,1.5H25.45A1.5,1.5,0,0,0,25.45,50.52Z"
+       id="path8"
+       style="fill:#ffffff" />
+  </g>
```
Comment (Pawan1005) [inline Bhavika/Bootstrap/bootstrap.min.css.xml:67]: make correct indentation

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**